pkg/util/netsh/netsh_test.go

```diff
@@ -0,0 +1,32 @@
+package netsh
```

Comment:
Need to add boilerplate copyright header here.

Reply:
😊Done!